volatility3/framework/plugins/windows/envars.py

```diff
@@ -58,13 +58,11 @@ def _get_silent_vars(self) -> List[str]:
         """
 
         values = []
-        kernel = self.context.modules[self.config["kernel"]]
 
         for hive in hivelist.HiveList.list_hives(
-            context=self.context,
+            self.context,

```

Comment:
```suggestion
            context=self.context,

```
No point leaving this one out...  5;)